src/backend/libs/efiling-account-client/src/main/java/ca/bc/gov/open/jag/efilingaccountclient/CsoAccountServiceImpl.java

```diff
@@ -50,4 +53,10 @@ public CsoAccountDetails getAccountDetails(String userGuid) {
         return csoAccountDetails;
```

Comment:
we should NestedEjbException_Exception otherwise it looks like the user does not have an account

Reply:
It is thrown!